samza-azure/src/main/java/org/apache/samza/system/azureblob/AzureBlobConfig.java

```diff
@@ -80,6 +80,12 @@ public class AzureBlobConfig extends MapConfig {
   public static final String SYSTEM_MAX_FLUSH_THRESHOLD_SIZE = SYSTEM_AZUREBLOB_PREFIX + "maxFlushThresholdSize";
   private static final int SYSTEM_MAX_FLUSH_THRESHOLD_SIZE_DEFAULT = 10485760;
 
+  // initialization size of in-memory OutputStream
+  // This value should be between SYSTEM_INIT_BUFFER_SIZE_DEFAULT and getMaxFlushThresholdSize() exclusive.
+  public static final String SYSTEM_INIT_BUFFER_SIZE = SYSTEM_AZUREBLOB_PREFIX + "initBufferSize.bytes";
```

Comment:
Does Samza initialize their configs in this format? Plz check the other examples and name it in similar fashion.

Reply:
I believe this is the correct way. I was following the guidelines from their "[Coding Guide](https://samza.apache.org/contribute/coding-guide.html)" _Configuration_. For reference, the rules I was applying:

* All configuration names that define a byte size must end with .bytes (e.g. foo.bar.bytes=1000).
* Configuration will always be defined as simple key/value pairs (e.g. a=b).
* When configuration is related, it must be grouped using the same prefix (e.g. job.container.count=1, yarn.container.memory.bytes=1073741824).
* All getter methods must be a camel case match with their configuration names (e.g. yarn.package.uri and getYarnPackageUri).
* Reading configuration should only be done in factories and main methods. Don’t pass Config objects around.